src/components/structures/LeftPanel.tsx

```diff
@@ -119,8 +119,12 @@ export default class LeftPanel extends React.Component<IProps, IState> {
         if (settingBgMxc) {
             avatarUrl = MatrixClientPeg.get().mxcUrlToHttp(settingBgMxc, avatarSize, avatarSize);
         }
+
         const avatarUrlProp = `url(${avatarUrl})`;
-        if (document.body.style.getPropertyValue("--avatar-url") !== avatarUrlProp) {
+        const existingValue = document.body.style.getPropertyValue("--avatar-url");
+        if (!avatarUrl && existingValue) {
```

Comment:
we can just leave this at `!avatarUrl`? there would be an existing value when the user removes their avatar